Source/diablo.cpp

```diff
@@ -627,6 +628,12 @@ static void LoadOptions()
 	sgOptions.Gameplay.bAutoEquipHelms = getIniBool("Game", "Auto Equip Helms", false);
 	sgOptions.Gameplay.bAutoEquipShields = getIniBool("Game", "Auto Equip Shields", false);
 	sgOptions.Gameplay.bAutoEquipJewelry = getIniBool("Game", "Auto Equip Jewelry", false);
+	getIniValue("Game", "Auto Equip Items", sgOptions.Gameplay.szAutoEquipItems, 99, "Weapon");
+	sgOptions.Gameplay.bAutoEquipWeapons |= IniFound(sgOptions.Gameplay.szAutoEquipItems, "Weapon");

```

Comment:
Perhaps there should be an `All` shorthand as well?

Reply:
Yes, `x |= y` is shorthand of  `x = x | y`.